Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "gtk-rs-examples"
 version = "0.0.1"
 authors = ["The Gtk-rs Project Developers"]
+autobins = false
```

Comment:
Why?

Reply:
IMHO to prevent attempt to build gio_futures_await.rs in future
```
warning: An explicit [[bin]] section is specified in Cargo.toml which currently
disables Cargo from automatically inferring other binary targets.
This inference behavior will change in the Rust 2018 edition and the following
files will be included as a binary target:

* D:\eap\rust\0\examples\src\bin\gio_futures_await.rs

This is likely to break cargo build or cargo test as these files may not be
ready to be compiled as a binary target today. You can future-proof yourself
and disable this warning by adding `autobins = false` to your [package]
section. You may also move the files to a location where Cargo would not
automatically infer them to be a target, such as in subfolders.

For more information on this warning you can consult
https://github.com/rust-lang/cargo/issues/5330

```

Reply:
@EPashkin: Exactly! Well found. ;)